roles/provision-sync-app-apb/tasks/main.yml

```diff
@@ -12,6 +12,8 @@
     spec_template_metadata_labels:
       app: sync-app
       service: sync-app
+    spec_template_metadata_annotations:
+      org.aerogear.metrics/plain_endpoint: '{{ sync_app_metrics_endpoint }}'
```

Comment:
so if the app doesn't have the `metrics` endpoint defined at all, it should not cause a problem for Prometheus, right?

Reply:
it will just show it as "Down".
hmm, actually, we can ideally not add the annotation for target auto discovery if it is not defined but I would prefer doing it later. it is not a super quick thing to do

Reply:
what I mean by "Down" is
* not Kubernetes health/readiness check Down
* Prometheus down